### PR TITLE
Ignore local config file

### PR DESCRIPTION
**Pull request by IronGauntlets**
## Description

Updates `.gitignore` to ignore local config files stored in `config/` and removes duplicated `.idea/` pattern.

## Types of changes

- Other (please describe): `.gitignore` update

## Testing

**Requires testing**: No

**Did you write tests??**: No

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,8 +5,8 @@
 build/
 coverage/*
 courtney/
-.idea/
```
Comment (stdevMac) [inline .gitignore:8]: re-add the `.idea/`, for those using Goland, definitely a must.

**Review by stdevMac — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (IronGauntlets) [reply .gitignore:8]: Its duplicated

Comment (IronGauntlets) [reply .gitignore:8]: Please look at `.gitignore:2`

Comment (stdevMac) [reply .gitignore:8]: Thanks!

**Review by stdevMac — APPROVED** [review]

**Review by adriantpaez — APPROVED** [review]

**Review by tshakalekholoane — APPROVED** [review]

**Pull request merged**